bin/sequenceserver

```diff
@@ -2,6 +2,7 @@
 require 'readline'
 require 'English'
 require 'slop'
+require 'sequenceserver'
```

Comment:
Requiring `sequenceserver` here will likely cause SequenceServer to default to development mode.